gluetool/tool.py

```diff
@@ -256,10 +257,11 @@ def _handle_failure(self, failure):
             except Exception:
                 # tripple error \o/
 
-                print >> sys.stderr
-                print >> sys.stderr, '!!! While logging an exception, another exception appeared !!!'
-                print >> sys.stderr, '    Giving up on everything...'
-                print >> sys.stderr
+                print(file=sys.stderr)
```

Comment:
What about to add `\n` to the beginning of next print?

Reply:
I replaced these with multiline strings - it is more readable in Python 2, with multiple `print` statements, in Python 3 it sucks.